Sources/Puree/LogStore/FileLogStore.swift

```diff
@@ -110,4 +110,8 @@ public class FileLogStore: LogStore {
         try? fileManager.removeDirectory(at: baseDirectoryURL)
         try? createCachesDirectory()
     }
+
+    private func encodeToBase16(_ string: String) -> String {
+        return string.data(using: .utf8)!.map { String(format: "%02hhx", $0) }.joined()
```

Comment:
https://en.cppreference.com/w/cpp/io/c/fscanf

According to this document.
using `h`, data is treated as u_int8, otherwise `hh` is `char*`.
It is better to use `hh` in this case.

Reply:
https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/Strings/Articles/formatSpecifiers.html